en/services/parameter.md

```diff
@@ -180,11 +189,29 @@ The drone may restart the sequence the `PARAM_VALUE` acknowledgment is not recei
 ### PX4
 
 PX4 implements the protocol in a way that is compatible with this specification.
+Only float and Int32 parameters are used.
+
 
 Source files:
 * [src/modules/mavlink/mavlink_parameters.cpp](https://github.com/PX4/Firmware/blob/master/src/modules/mavlink/mavlink_parameters.cpp)
 * [src/modules/mavlink/mavlink_parameters.h](https://github.com/PX4/Firmware/blob/master/src/modules/mavlink/mavlink_parameters.h)
 
 ### ArduPilot
 
-TBD
+ArduPilot implements an incompatible version of this protocol.
```

Comment:
@tridge I have put you as the reviewer for this because you're probably the only person who can confirm that I have properly captured the differences between ArduPilot implementation and the spec. This is important because it will make it easier for GCS developers to write code that talks to ArduPilot.